src/main/java/duke/Deadline.java

```diff
@@ -0,0 +1,46 @@
+package duke;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
+/**
+ * A task with a specified deadline.
+ */
+public class Deadline extends Task {
```

Comment:
can't find much coding standard issues not already found by other reviewer. Really clear code with good use of OOP!